### PR TITLE
- #76 update the predicate syntax for ruby and css

### PR DESCRIPTION
**Pull request by TravonteD**

### CODE REVIEW

File: queries/ruby/highlights.scm [hunk 1]
```diff
@@ -88,7 +88,7 @@
 (keyword_parameter (identifier) @parameter)
 
 ((identifier) @function
- (is-not? local))
+ (#is-not? local))
```
Comment (vigoux) [inline queries/ruby/highlights.scm:91]: I just see this predicate, and it is not supported (yet)

**Comment by vigoux** [comment]
Thanks for updating these queries !

File: queries/ruby/highlights.scm (hunk 1 above)
Comment (TravonteD) [reply queries/ruby/highlights.scm:91]: Do we want to remove it for now?

Comment (vigoux) [reply queries/ruby/highlights.scm:91]: Yes maybe, more like commenting it as a TODO for me for example

**Pull request merged**